Gemfile.lock

```diff
@@ -662,7 +677,7 @@ GEM
     xml-simple (1.1.4)
     xpath (2.0.0)
       nokogiri (~> 1.3)
-    zeus (0.15.4)
+    zeus (0.15.13)
```

Comment:
I am not using zeus so not sure what is the problem. And I've heard that @oeoeaio wants to remove it from `Gemfile` so just upgraded it in `Gemfile.lock`